manim/mobject/svg/tex_mobject.py

```diff
@@ -1,9 +1,10 @@
 r"""Mobjects representing text rendered using LaTeX.
 
+.. note::
+   Just as you can use :class:`~.Text` (from the module :mod:`~.text_mobject`) to add text to your videos, you can use :class:`~.Tex` and :class:`~.MathTex` to insert LaTeX.
```

Comment:
I don't think the module name is necessary. Same for the other change also.
```suggestion
   Just as you can use :class:`~.Text` to add text to your videos, you can use :class:`~.Tex` and :class:`~.MathTex` to insert LaTeX.
```

Reply:
I think it is good to have, as the text_mobject module is right now really difficult to find in the docs.
And when one wants to use tex, one might probably also see some example of how to use text.
Also, it makes it more clear, that tex is not part of of the text_mobject